cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/datastore/ForceInsertTest.java

```diff
@@ -84,14 +86,20 @@ public void notification_forceinsertWithAttachments() throws Exception {
 
         ArrayList<String> revisionHistory = new ArrayList<String>();
         revisionHistory.add(doc1_rev1.getRevision());
+        revisionHistory.add("2-revision");
+
+        // We need to create a "clean" document for force insert of the attachment
+        // manager will add the attachments to seq 1 instead of seq 2.
```

Comment:
This comment needs to be deleted as I don't think the attachments are on seq 1 and we don't really have any indication in this code what is seq 1 and what is seq 2.

Besides that I think it would be worth saying that we are force inserting 2-revision which has attachments and what we are expecting a notification for.